src/barkhaPJun20/Bank.java

```diff
@@ -0,0 +1,88 @@
+package barkhaPJun20;
+
+public class Bank {
+	int debitCount=0;
+	int creditCount=0;
+	int balanceCount=0;
+	
+	static int overallDebitCount=0;
+	static int overallCreditCount=0;
+	static int overallBalanceCount=0;
+	
+	int debitAmnt;
+	int creditAmnt;
+	int currentBalance;
+	
+	
+	int user;
+	
+	void setInitialAmnt(int uID,int Amnt){
+		user=uID;
+		currentBalance=Amnt;
+	}
+	
+	void debitAmount( int dAmnt){
+		debitAmnt=dAmnt;
+		currentBalance=currentBalance-debitAmnt;
```

Comment:
debitAmnt=dAmnt; This line is not needed. You should directly use dAmnt variable.
currentBalance=currentBalance-dAmnt;

Reply:
done